crates/krator/examples/moose.rs

```diff
@@ -528,6 +525,21 @@ Running moose example. Try to install some of the manifests provided in examples
     
     "#
     );
-    runtime.start().await;
+
+    // New API does not currently support Webhooks, so use legacy API if enabled.
```

Comment:
I think the webhooks will always be optional since they bring in some dependencies. The `OperatorRuntime` API will probably be removed once I get webhooks working with the new API. I think Krator is still a ways away from 1.0, so that will probably happen before 1.0.